src/moonstone-samples/lib/DataGridListSample.css

```diff
@@ -34,6 +33,11 @@
     left: auto;
 }
 
+.enyo-locale-right-to-left .moon-gridlist-imageitem.horizontal-gridList-image-item .moon-overlay-container {
```

Comment:
I think I understand the intent, but the selection icon is not centered on the image it overlays (in RTL). We should probably fix the positioning of the selection icon when the item type is set to "HorizontalItem", as well. Also, could you fix the merge conflict - I believe this is due to us changing the filename from `DataGridListSample.css` to `DataGridListSample.less`.